test/node/parseIDLFile-helper.js

```diff
@@ -0,0 +1,87 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+global.parseIDLFileTest = function(data) {
+  describe(data.description, function() {
+    var execSync = require('child_process').execSync;
+    var originalTimeout;
+    var LocalGitRunner;
+    var outputBox;
+    var errorBox;
+
+    afterAll(function() {
+      execSync(`/bin/rm -rf "${data.localRepositoryPath}"`);
+    });
+
+    beforeEach(function() {
+      foam.CLASS({
```

Comment:
Whoops, I fixed that previously but I forgot to push. Duplicate should be gone now